### DISCUSSION TITLE
一根胡萝卜引发的血案 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#472] [General]
# 一根胡萝卜引发的血案 - Miao Yu | 于淼

https://yufree.cn/cn/2024/08/06/case-carrot/

**Comment by achuanya** [comment]
我小时候摔着把一只门牙磕断了一半，去医院镶嵌了一个，晚上吃苹果连假牙吃了...一晃十几年了

**Reply by yufree** [reply]
是乳牙吗？要是恒牙就比较尴尬了。

**Comment by yihui** [comment]
美帝的牙医和保险（包括普通医疗保险）真是无力吐槽。我已经五年没去看过牙医了，就是不想听他们一堆废话各种劝治疗，听了神烦。就比如智齿，每次看牙医都劝我把它们全拔了。那种长歪了智齿我觉得拔了可以理解，但美帝牙医好像觉得拔智齿是天经地义的。

生活在美帝只能祈祷身体零部件别出毛病、别生病，因为看医生实在是令人非常不爽，效率低下、慢得要死、治疗不是极端保守（回家自己熬）就是极端激进（过度治疗）。这也算是我们平日要坚持锻炼的一大动力吧，哈哈，反正也指望不上医生。

**Reply by yufree** [reply]
在美国牙医这边，他们不把智齿当成牙，单纯当成风险，保险覆盖后单纯拔智齿要不是我这种长歪了的也不贵，但会推销一堆保险不报的东西，特别是全麻，你不问也不告诉你本来就有局麻，主打一个信息差赚钱。不过有保险一年去洗一次牙也可以，但确实他们会推销一大堆东西很烦，有的不推销东西，但让你一年来四次，因为保险只报两次，感觉牙医那边把这些规则漏洞算是都玩明白了，而且还以一种我是专家听我的的态度。

另外就是人口基数很关键，我有朋友需要做一个手术只能到纽约做，而且大夫也不熟练，而国内省级专科医院就能做，单纯就是病人多手熟，毕竟有些手术属于经验活，同样发病率，国内大夫手术多，累积经验要比这边预约制多，当然另一方面国内大夫不如这边待遇好闲暇多。

反正现在大语言模型平替家庭医生，医疗这块平时还真就是得靠自己，真要是到了需要进医院的场景，听天由命吧。

**Comment by tcgriffith** [comment]
不知道发了UBI楼主会不会还是选择回国拔牙 :)

**Reply by yufree** [reply]
发不发应该都会回国拔，主要我周围这些牙医问一圈都一路货色，全是全麻四颗起步，亏我保险还按最贵的交还给算错过账单搞的差点让催收公司上门。要说换个地方也许能碰上能商量的牙医，但不费那劲了，国内直接去口腔医院15分钟搞定，这边又是预约又是找人接送一堆事，美国这医疗保险体系早晚得大改，现在是利益集团绑架了。

**Reply by tcgriffith** [reply]
一个医疗，一个军工，百万漕工衣食所系 难改啊

**Comment by iotanano** [comment]
国内智齿被口腔内（补牙）、外（拔牙）科医生当成风险、建议拔掉，主要是：
1. 如果智齿没有长歪，清洁困难，牙刷牙线很难清理干净它，容易龋坏/蛀牙、引发牙龈炎症。门诊时遇到过 60 多岁的老爷爷、老太太来拔智齿的。
2. 如果歪了，阻生牙，会挤到前面的大牙，连累大牙一起坏。

然而，并不是每个医生都想拔掉智齿，正畸科医生就喜欢它，毕竟是原装材料 :) 

俺两颗大牙根冠治疗后没保护好，碎的厉害无法修复，正好喜欢啃硬的东西、牙根长，相邻两颗智齿一颗正、一颗半萌生的阻生牙，就拉它俩顶上。

拔掉大牙残冠后（外科医生叹气，难拔），长得正的直接牵引过来，没完全长出来的阻生牙引导长出来再牵引，搭配后续全口钢丝正畸，好使。

智齿斗争血泪经验分享：
- 牙龈发炎时、月经期间、高血压/心脏病患者、有凝血问题的，不给拔牙。
- 低浓度双氧水（3% 浓度，再稀释）也可以缓解智齿炎症。
- 拔智齿须拍片，有的综合性医院可能还要查血。
- 如果怕痒，可以提前提醒医生不要把手术器械放在自己胸口（有的医生会有这种坏习惯）。医生非常怕口腔手术时患者笑/不稳定。
- 拔智齿，如果口腔外科人多，可以去神经外科拔，那边人少。

**Reply by yufree** [reply]
额，还有让智齿再就业的做法啊，不过这个牵引正畸过程想想都疼。

**Reply by iotanano** [reply]
- 拔牙后恢复期禁酒。因为酒精会影响骨骼恢复。
- 拔牙后最害怕的是[干槽症](https://www.mayoclinic.org/zh-hans/diseases-conditions/dry-socket/symptoms-causes/syc-20354376)。见势不妙赶紧回去找医生。
- 打算备孕的朋友，一般建议提前解决牙齿问题（拔牙补牙啥的）。

当然，牙齿健康状况除了后天维护，也非常受到个人先天因素影响。
- 牙齿先天条件好非常省钱。
- 不那么好的，为了保护健康与荷包，只能勤刷牙（干刷）、用牙线（可以早点发现邻面龋）、定期洗牙（清理软垢与牙结石）、及时补牙（省钱而且能走医保）、小心隐裂（不定时炸弹）、避免摄入碳酸饮料/酒精（保护牙釉质、血管、骨骼）、尽量避免冷热刺激（保护牙本质）……

**Reply by iotanano** [reply]
痛倒是不痛的，进口麻醉针都很细，牙槽骨神经少，支抗钉也没啥痛感。牵引需要 1-2 年时间，温水煮青蛙。

比起痛，可能主要是缠皮筋麻烦哈哈。

**Reply by yufree** [reply]
虽然但是，我已经决定吃自己喜欢的东西，过短命的人生了 -- 坂田鱼食
